migrations/migrations/20210618203322-user.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('users', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      lastName: {
```

Comment:
table fields should be in `snake_case`.

Reply:
This was about a confusion in the tech-guides
```
module.exports = (sequelize, DataTypes) => {
  const User = sequelize.define(
    'users',
    {
      mail: { type: DataTypes.STRING },
      fullName: { type: DataTypes.STRING },
      phone: { type: DataTypes.STRING }
    },
    {
      timestamps: false
    }
  );
  return User;
};

```